api/src/pcapi/core/external/attributes/api.py

```diff
@@ -287,12 +288,22 @@ def get_pro_attributes(email: str) -> models.ProAttributes:
         .all()
     )
 
+    if has_collective_offers:
```

Comment:
Attention ici, tu utilises `has_collective_offers` qui est calculé en recherchant par adresse email de compte pro, comme condition alors que tu calcules `has_collective_offers_marseille_en_grand` à partir des lieux collectés en recherchant par `venue.bookingEmail`. Je crains qu'on puisse se retrouver avec des incohérences, car `has_collective_offers` sera `False` si l'email est utilisé uniquement dans `venue`.

En intuition rapide, je me dis que la condition pour faire la requête pourrait être sur quelque chose comme `any(venue.adageId for venue in venues)`.

Je te laisse vérifier.

Reply:
En fait le critère `any(venue.adageId for venue in venues)` n'est pas suffisant => il peut y avoir un lieux qui réaliser des projets collectif (pour marseille en grand) en dehors de l'établissement scolaire. Il faut donc a minima faire une première requete pour savoir si les lieux en questions ont été utilisé pour des offres collectives. 

Puisqu'on doit faire une requête dans tous les cas. Je propose de supprimé la condition.

Reply:
Je ne comprends pas bien : si le lieu n'a pas d'`adageId`, alors il ne peut pas proposer d'offre collective, donc n'aura jamais une offre reliée à un établissement scolaire relié lui-même au programme MEG. Mais il me manque peut-être des infos métier, qu'ai-je loupé ?

Reply:
tu as raison -> il y a forcément un lieu de rattachement. Ne pas tenir compte de ma remarque précédente.